### PR TITLE
Add jitsi-videobridge plugin

### PR DESCRIPTION
**Pull request by brknkfr**
Simple multigraph plugin with one graph for current session infos and one
graph for total conferences created.

### CODE REVIEW

File: plugins/jitsi-videobridge/jitsi-videobridge [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+    autoconf)
+        if [ -x /usr/bin/curl ]; then
+            if [ -x /usr/bin/jq ]; then
+                echo "yes"
+                exit 0
+            else
+                echo "no (jq not found)"
+                exit 0
+            fi
```
Comment (sumpfralle) [inline plugins/jitsi-videobridge/jitsi-videobridge:58]: For a perfectly self-configuring plugin you should add a test for an empty result of `curl "$COLIBRI_URL"` here. In this case you can also use the family `auto`.

btw: the `/stats` part of the URL path is probably based on the API of the jitsi service. Maybe you want to let the configurable part of the URL only be `http://localhost:8080/colibri` and add the `/stats` path in the plugin. This would allow the plugin to re-adjust the requested path, in case of changes in the jitsi API.

But this is a matter of taste, I guess.

**Comment by sumpfralle** [comment]
Great - thanks!

One last thing: if you do not mind, then I would like to rename `plugins/jitsi_videobridge/jitsi_videobridge` to `plugins/jitsi/jitsi_videobridge`. Or do you think, plugins for the videobridge component should be kept separete from other jitsi-related plugins?
